src/lib.rs

```diff
@@ -4,7 +4,7 @@
 //! which allow very fast iteration
 //! on sparse data structures.
 
-#![deny(missing_docs)]
+// #![deny(missing_docs)]
```

Comment:
temporary?

Reply:
Yeah, gonna write docs when I am sure about the API.